### DISCUSSION TITLE
Zoom in changing cameras

### DISCUSSION THREAD

**Discussion by PalidoPardal** [ramokz/phantom-camera#245] [Question / Help Wanted]
When i change the Phantom Camera 2D (with set priority in the camera to change) the zoom isn't changed.

**Comment by ramokz** [comment]
Not sure what the cause of that would be — as it works in the example scenes.
Can you share a screenshot of your scene composition, a video capture, or a MRP (minimum reproducible project) to look at?

**Comment by PalidoPardal** [comment]
![Captura](https://github.com/ramokz/phantom-camera/assets/115578549/54bfb2b0-4453-4f72-bb7d-010634c61e41)
The camera in the screenshoot is the normal camera (with 2/2 zoom) and each player have a camera with 6/6 zoom and when i change the priority to the camera of one of the players the camera change to change to it but without modifying its zoom. (The version of godot is 4.2)

**Reply by PalidoPardal** [reply]
![image](https://github.com/ramokz/phantom-camera/assets/115578549/bbd0be6b-0816-497f-be78-e88b36df06e2)
This is the script to change to the player camera (the priority of the general camera is 10)

**Reply by ramokz** [reply]
Can't see anything obviously wrong with your setup.
I'm guessing you're using the latest version of the addon?

**Reply by PalidoPardal** [reply]
I use the latest (0.6.4). The strange thing is that with the example scene it works normally but with mine it doesn't.

**Reply by PalidoPardal** [reply]
![image](https://github.com/ramokz/phantom-camera/assets/115578549/424cfc73-ef8b-45bb-af89-1975c6205c38)
I find this on the Phantom Controller code , this shouldn't affect anything, right?

**Reply by ramokz** [reply]
No, that's just a note to say that the name of the variable couldn't be called `zoom` due to a side effect bug that occurred.

**Reply by ramokz** [reply]
Might be worth trying unchecking `Tween on Load` on all the PCam2D instances and see if that does anything

**Reply by PalidoPardal** [reply]
ok , thanks. I try to compare the example scene with my scene to check if there is an error

**Reply by ramokz** [reply]
You're also welcome to share a project sample. Happy to take a look as well.
